pom.xml

```diff
@@ -0,0 +1,577 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+    http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.openmicroscopy</groupId>
+  <artifactId>ome-common</artifactId>
+  <version>6.0.0-SNAPSHOT</version>
```

Comment:
One question here is whether we should not be conservative and target this as `5.3.0-SNAPSHOT`. As far as I know, our first target is a Maven release with no API change and for consumers the major change is the usage of a different `groupId/artifactId`  (and only if they are declaring it directly in their POM).